src/backend/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": false,
+    "noEmit": true,
+    "typeRoots": ["types", "node_modules/@types"]
```

```suggestion
    "target": "es2019",
    "lib": ["es2020"],
    "resolveJsonModule": true,
    "allowJs": true,
    "checkJs": false,
    "noEmit": true,
    "typeRoots": ["types", "node_modules/@types"]
```

`resolveJsonModule` ensures that requiring `.json` files still works (off by default)
`lib` tells Typescript what type definitions to include. We run Node v12.16.3 on mobile, which supports all ES2020 _modules_ (see https://stackoverflow.com/a/59787575/3071863)
`target` is not actually used, since we don't use Typescript to compile, but just adding it in case we turn that on. Node v12 only supports 100% es2019 syntax, not all es2020.